README.md

```diff
@@ -1,3 +1,86 @@
-# Developer Rubric Discord Bot
+<div id="top"></div>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://avatars.githubusercontent.com/u/31629704?s=200&v=4">
+    <img src="./readmeAssets/logo.svg" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">Developer Rubric Discord Bot</h3>
+
+  <p align="center">
+    <a href="">View Demo</a>
+    ·
+    <a href="https://github.com/Semalab/developer-rubric-discord-bot/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=bug_report.yml&title=%5BBUG%5D+-+">Report Bug</a>
+    ·
+    <a href="https://github.com/Semalab/developer-rubric-discord-bot/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=feature_request.yml&title=%5BFEAT%5D+-+">Request Feature</a>
+  </p>
+</div>
+
+## About The Project
+
+<!-- [![Developer Rubric Discord Bot][product-screenshot]](https://example.com) -->
 
 This is a Discord bot which allows you to assess your skills with our [developer rubric](https://github.com/Semalab/developer-rubric) directly on Discord.
+
+Here's what developer rubric is:
+
+- A self-assessment tool you can use to measure your current place in your development journey.
+- It provide The markdown rubric, The JSON rubric, and The CSV rubric
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+### Installation
+
+_How to run the discord bot locally._
+
+1. Setting up discord bot:
+
+- Login to the Discord Developer portal and create a new Application.
+- Add a bot to the Application after selecting "Bot" from the left-side panel.
+- Copy the bot's token.
+- Go to the Oauth2 tab > URL Generator and select both bot and applications.commandss scopes, along with the - Administrator permission. Copy the URL generated and invite the bot in your server.
+- Copy the guild id of the server you invited the bot into
+
+2. Clone the repo
+   ```sh
+   git clone https://github.com/Semalab/developer-rubric-discord-bot.git
```

Comment:
I think you should use `YOUR_USERNAME_HERE` so that people clone their fork instead of the original repository.
```suggestion
   git clone https://github.com/YOUR_USERNAME_HERE/developer-rubric-discord-bot.git
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix